mesonbuild/templates/cythontemplates.py

```diff
@@ -0,0 +1,67 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2019 The Meson development team
+
+from __future__ import annotations
+
+from mesonbuild.templates.sampleimpl import FileImpl
+
+import typing as T
+
+
+hello_pyx_template = '''print("This is project {project_name}.")'''
```

Comment:
File should end with `\n`.

```suggestion
hello_pyx_template = '''print("This is project {project_name}.")
'''
```

Also double quotes -> single quotes?